main/src/ca/uwaterloo/flix/language/phase/unification/EffUnification3.scala

```diff
@@ -290,15 +311,18 @@ object EffUnification3 {
       *     false for `ef`)
       *   - `getAtoms(Indexable.Aef[Error]) = Set.empty`
       */
-    def getAtoms(t: Type)(implicit scope: Scope, renv: RigidityEnv): Set[Atom] = t match {
-      case Type.Var(sym, _) if renv.isRigid(sym) => Set(Atom.VarRigid(sym))
-      case Type.Var(sym, _) => Set(Atom.VarFlex(sym))
-      case Type.Cst(TypeConstructor.Effect(sym), _) => Set(Atom.Eff(sym))
-      case Type.Cst(TypeConstructor.Error(id, _), _) => Set(Atom.Error(id))
+    def getAtoms(t: Type)(implicit scope: Scope, renv: RigidityEnv): SortedSet[Atom] = t match {
+      case Type.Var(sym, _) if renv.isRigid(sym) => SortedSet(Atom.VarRigid(sym))
+      case Type.Var(sym, _) => SortedSet(Atom.VarFlex(sym))
+      case Type.Cst(TypeConstructor.Effect(sym), _) => SortedSet(Atom.Eff(sym))
+      case Type.Cst(TypeConstructor.Error(id, _), _) => SortedSet(Atom.Error(id))
       case Type.Apply(tpe1, tpe2, _) => getAtoms(tpe1) ++ getAtoms(tpe2)
       case Type.Alias(_, _, tpe, _) => getAtoms(tpe)
-      case assoc@Type.AssocType(_, _, _, _) => getAssocAtoms(assoc).toSet
-      case _ => Set.empty
+      case assoc@Type.AssocType(_, _, _, _) => getAssocAtoms(assoc) match {
```

Comment:
Is there no `to(SortedSet)`?

Reply:
It could be `SortedSet.from(getAssocAtoms(assoc))` im pretty sure

Reply:
PR welcome